backend/src/resolvers/bill.ts

```diff
@@ -1,22 +1,22 @@
 import { get } from 'http';
```

Comment:
This get is never used no?

Reply:
cameron said to delete 👍 it was deleted in the new push